src/Assignment-5/assignment5.html

```diff
@@ -1,20 +1,45 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<title></title>
 	<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
 	<meta http-equiv="Pragma" content="no-cache" />
 	<meta http-equiv="Expires" content="0" />
 	<script src="bundle.js" type="text/javascript"></script>
+
+	<link rel="stylesheet" href="assignment5.css">
+
 </head>
 
 <body>
 	<div id="container">
+		<!-- Start your code from here -->
+		<div class="card">
+			<div class="content1">
+				<div class="div1"><img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1145795/plant-1.png"></div>
+				<div>
+					<h1>A leafy one</h1>
+					<p>Plants are predominantly photosynthetic eukaryotes of the kingdom Plantae. Historically, the
+						plant kingdom encompassed all living things.</p>
+					<div class="btn">SO LEAPY</div>
+				</div>
+			</div>
 
-	<!-- Start your code from here -->
-	
+			<div class="content2">
+				<div class="div1"><img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1145795/plant-4.png"></div>
```

Comment:
i guess your image 2 is not in accordance with the document, please check the word document and make necessary changes